src/core/src/engine.rs

```diff
@@ -29,10 +29,13 @@ use serde_json;
 use std::time::Duration;
 use std::time::Instant;
 use enclave_sim::*;
-use std::collections::{HashMap};
+use std::collections::HashMap;
 use consensus_state::*;
-use consensus_state_store::{ConsensusStateStore, InMemoryConsensusStateStore};
+use consensus_state_store::ConsensusStateStore;
 use poet2_util;
+use database::config;
+use database::lmdb;
```

Comment:
Please consider refactoring lmdb instance into consensus state in next iteration.

Reply:
yes